### PR TITLE
feat: icky 879

### PR DESCRIPTION
**Pull request by kibo-chandradeeptalaha**

### CODE REVIEW

File: components/b2b/seller/AccountsTable/AccountsTable.tsx [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+                        {accountName}
+                      </Typography>
+                    </TableCell>
+                    <TableCell component="td" scope="row" sx={{ whiteSpace: 'break-spaces' }}>
+                      <Typography variant="body2" data-testid={`quote-name`}>
+                        {email}
+                      </Typography>
+                    </TableCell>
+                    <TableCell component="td" scope="row">
+                      <Typography variant="body2" data-testid={`quote-expirationDate`}>
```
Comment (Kibo-TriptiV) [inline components/b2b/seller/AccountsTable/AccountsTable.tsx]: change the data-testid everywhere.

File: components/b2b/seller/AccountsTable/AccountsTable.tsx [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+import { AccountsTableStyles } from './AccountsTable.styles'
+import { KiboPagination, SearchBar } from '@/components/common'
+import { QuotesFilterDialog } from '@/components/dialogs'
+import { useModalContext } from '@/context'
+import { useDebounce } from '@/hooks'
+import { addressGetters } from '@/lib/getters'
+// import { buildQuotesFilterParam } from '@/lib/helpers'
+// import { QuoteFilters,  } from '@/lib/types'
+
+import { QueryQuotesArgs } from '@/lib/gql/types'
```
Comment (Kibo-TriptiV) [inline components/b2b/seller/AccountsTable/AccountsTable.tsx]: use type

File: components/b2b/seller/AccountsTable/AccountsTable.tsx [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+import getConfig from 'next/config'
+import { useTranslation } from 'next-i18next'
+
+import { AccountsTableStyles } from './AccountsTable.styles'
+import { KiboPagination, SearchBar } from '@/components/common'
+import { QuotesFilterDialog } from '@/components/dialogs'
+import { useModalContext } from '@/context'
+import { useDebounce } from '@/hooks'
+import { addressGetters } from '@/lib/getters'
+// import { buildQuotesFilterParam } from '@/lib/helpers'
```
Comment (Kibo-TriptiV) [inline components/b2b/seller/AccountsTable/AccountsTable.tsx]: remove commented code

File: hooks/custom/useHandleB2BContacts/useHandleB2BContacts.ts [hunk 4]
```diff
@@ -0,0 +1,38 @@
+import { useState } from 'react'
+
+import { QueryQuotesArgs } from '@/lib/gql/types'
```
Comment (Kibo-TriptiV) [inline hooks/custom/useHandleB2BContacts/useHandleB2BContacts.ts]: use type everywhere

File: src/pages/my-account/b2b/seller/manage-quotes/index.tsx [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+
+  return (
+    <>
+      <MobileB2BLayout
+        headerText={t('create-a-quote')}
+        backText={activeBreadCrumb?.backText}
+        onBackClick={onBackClick}
+      />
+      <Box pb={2}>
+        <Typography variant="h5">Create a quote for an account from the list below</Typography>
```
Comment (Kibo-TriptiV) [inline src/pages/my-account/b2b/seller/manage-quotes/index.tsx:89]: please localize the text

File: src/pages/my-account/b2b/seller/manage-quotes/index.tsx [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+      </Box>
+      <AccountsTable
+        b2bContacts={b2bContacts}
+        setB2BContactsSearchParam={handleB2BContactsSearchParam}
+      />
+      <Box py={2}>
+        <Typography variant="h2">All Quotes</Typography>
```
Comment (Kibo-TriptiV) [inline src/pages/my-account/b2b/seller/manage-quotes/index.tsx:96]: please localize the text

File: components/b2b/seller/AccountsTable/AccountsTable.tsx [hunk 7]
```diff
@@ -0,0 +1,223 @@
...
+    {
+      field: 'zipcode',
+      headerName: t('zip-code'),
+    },
+  ]
+
+  const [searchTerm, setSearchTerm] = useState('')
+  const debouncedTerm = useDebounce(searchTerm, publicRuntimeConfig.debounceTimeout)
+
+  // const handleFilterAction = (filters: QuoteFilters) => {
```
Comment (Kibo-TriptiV) [inline components/b2b/seller/AccountsTable/AccountsTable.tsx:82]: remove commented codes

**Review by Kibo-TriptiV — APPROVED** [review]
please address minor review comments

File: src/pages/my-account/b2b/seller/manage-quotes/index.tsx (hunk 6 above)
Comment (kibo-chandradeeptalaha) [reply src/pages/my-account/b2b/seller/manage-quotes/index.tsx:96]: Implemented in icky-893 PR

File: components/b2b/seller/AccountsTable/AccountsTable.tsx (hunk 7 above)
Comment (kibo-chandradeeptalaha) [reply components/b2b/seller/AccountsTable/AccountsTable.tsx:82]: Implemented in icky-893 PR

File: src/pages/my-account/b2b/seller/manage-quotes/index.tsx (hunk 5 above)
Comment (kibo-chandradeeptalaha) [reply src/pages/my-account/b2b/seller/manage-quotes/index.tsx:89]: Implemented in icky-893 PR

File: hooks/custom/useHandleB2BContacts/useHandleB2BContacts.ts (hunk 4 above)
Comment (kibo-chandradeeptalaha) [reply hooks/custom/useHandleB2BContacts/useHandleB2BContacts.ts]: Done

File: components/b2b/seller/AccountsTable/AccountsTable.tsx (hunk 3 above)
Comment (kibo-chandradeeptalaha) [reply components/b2b/seller/AccountsTable/AccountsTable.tsx]: Implemented in icky-893 PR

File: components/b2b/seller/AccountsTable/AccountsTable.tsx (hunk 2 above)
Comment (kibo-chandradeeptalaha) [reply components/b2b/seller/AccountsTable/AccountsTable.tsx]: Done

File: components/b2b/seller/AccountsTable/AccountsTable.tsx (hunk 1 above)
Comment (kibo-chandradeeptalaha) [reply components/b2b/seller/AccountsTable/AccountsTable.tsx]: Done

**Pull request merged**